server/src/controllers/analytics_controller.ts

```diff
@@ -59,27 +59,42 @@ function onTaskResult(taskResult: TaskResult) {
   }
 }
 
-async function onDetect(detectionResult: DetectionResult) {
+/**
+ * Processes detection result from analytics` DETECT message
+ * Returns IDs of segments inserted into DB if there was no merging
```

Comment:
```suggestion
 * @returns IDs of segments inserted into DB if there was no merging
```

more about this: https://github.com/Microsoft/tsdoc
https://github.com/Microsoft/tsdoc